examples/try_everything.py

```diff
@@ -20,7 +19,7 @@ def test_sync(request):
 
 @app.route("/dynamic/<name>/<i:int>")
 def test_params(request, name, i):
-    return response.text("yeehaww {} {}".format(name, i))
+    return response.text(f"yeehaww {name} {i}")
```

Comment:
Yes, Ruff auto-changed old formatting to f strings, and also removed the f of some string literals that didn't need to be f-strings. I think it also has an option (not yet enabled) to combine adjacent literals to avoid `"previously on one line " "second line now on same line"` being oddly two separate literals once put to same line in formatting.

Reply:
All format/% were converted to f strings. Useless f on string literals with no `{}` were similarly removed. Various other updates of similar fashion were performed on other stuff than strings as well. These come from pyupdate UP rules.